### PR TITLE
Enablement of recent version of HIP-VS plugin in vcxproj files:

### PR DESCRIPTION
**Pull request by hchevanne**
### Enablement of recent version of HIP-VS plugin in vcxproj files (39 examples):
Applications/floyd_warshall/**floyd_warshall_vs2019.vcxproj**
HIP-Basic/assembly_to_executable/**assembly_to_executable_vs2019.vcxproj**
HIP-Basic/bandwidth/**bandwidth_vs2019.vcxproj**
HIP-Basic/bit_extract/**bit_extract_vs2019.vcxproj**
HIP-Basic/cooperative_groups/**cooperative_groups_vs2019.vcxproj**
HIP-Basic/device_globals/**device_globals_vs2019.vcxproj**
HIP-Basic/device_query/**device_query_vs2019.vcxproj**
HIP-Basic/dynamic_shared/**dynamic_shared_vs2019.vcxproj**
HIP-Basic/events/**events_vs2019.vcxproj**
HIP-Basic/gpu_arch/**gpu_arch_vs2019.vcxproj**
HIP-Basic/inline_assembly/**inline_assembly_vs2019.vcxproj**
HIP-Basic/llvm_ir_to_executable/**llvm_ir_to_executable_vs2019.vcxproj**
HIP-Basic/matrix_multiplication/**matrix_multiplication_vs2019.vcxproj**
HIP-Basic/module_api/**module_api_vs2019.vcxproj**
HIP-Basic/moving_average/**moving_average_vs2019.vcxproj**
HIP-Basic/multi_gpu_data_transfer/**multi_gpu_data_transfer_vs2019.vcxproj**
HIP-Basic/occupancy/**occupancy_vs2019.vcxproj**
HIP-Basic/opengl_interop/**opengl_interop_vs2019.vcxproj**
HIP-Basic/runtime_compilation/**runtime_compilation_vs2019.vcxproj**
HIP-Basic/saxpy/**saxpy_vs2019.vcxproj**
HIP-Basic/shared_memory/**shared_memory_vs2019.vcxproj**
HIP-Basic/static_host_library/library/**libhip_static_host_vs2019.vcxproj**
HIP-Basic/static_host_library/**static_host_library_vs2019.vcxproj**
HIP-Basic/streams/**streams_vs2019.vcxproj**
HIP-Basic/texture_management/**texture_management_vs2019.vcxproj**
HIP-Basic/vulkan_interop/**vulkan_interop_vs2019.vcxproj**
HIP-Basic/warp_shuffle/**warp_shuffle_vs2019.vcxproj**
Libraries/exampleLibraryTemplate/example_template/**example_template_vs2019.vcxproj**
Libraries/hipCUB/device_radix_sort/**device_radix_sort_vs2019.vcxproj**
Libraries/hipCUB/device_sum/**device_sum_vs2019.vcxproj**
Libraries/rocPRIM/block_sum/**block_sum_vs2019.vcxproj**
Libraries/rocPRIM/device_sum/**device_sum_vs2019.vcxproj**
Libraries/rocRAND/simple_distributions_cpp/**simple_distributions_cpp_vs2019.vcxproj**
Libraries/rocThrust/device_ptr/**device_ptr_vs2019.vcxproj**
Libraries/rocThrust/norm/**norm_vs2019.vcxproj**
Libraries/rocThrust/reduce_sum/**reduce_sum_vs2019.vcxproj**
Libraries/rocThrust/remove_points/**remove_points_vs2019.vcxproj**
Libraries/rocThrust/saxpy/**saxpy_vs2019.vcxproj**
Libraries/rocThrust/vectors/**vectors_vs2019.vcxproj**

### CODE REVIEW

File: Applications/floyd_warshall/floyd_warshall_vs2019.vcxproj [hunk 1]
```diff
@@ -30,19 +28,20 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>HIP</PlatformToolset>
+    <PlatformToolset>HIP_clang</PlatformToolset>
```
Comment (saadrahim) [inline Applications/floyd_warshall/floyd_warshall_vs2019.vcxproj:31]: HIP_clang is an internal term and should not be used.

Comment (hchevanne) [reply Applications/floyd_warshall/floyd_warshall_vs2019.vcxproj:31]: Where does this statement come from? HIP_clang is the name used by the VS extension and which has been approved by AMD legal.
I'm puzzled.

Comment (saadrahim) [reply Applications/floyd_warshall/floyd_warshall_vs2019.vcxproj:31]: The compiler team refers to hip_clang as an informal term. 
Hip-clang: informal term for the amdclang compiler.

**Comment by hchevanne** [comment]
It's really too bad, "hip_clang" appears everywhere in the HIP-SDK, "amdclang" does not appear anywhere:
herve@SN-26508[Ubuntu 20.04.5 LTS]:/mnt/c$ grep -r -i **amdclang** /mnt/c/Program\ Files/AMD/ROCm/5.4/| wc -l
0
herve@SN-26508[Ubuntu 20.04.5 LTS]:/mnt/c$ grep -r -i **hip_clang** /mnt/c/Program\ Files/AMD/ROCm/5.4/| wc -l
151

**Comment by Snektron** [comment]
Implemented in #20

**Pull request closed without merging**